app/models/manageiq/providers/redhat/infra_manager/event_catcher/runner.rb

```diff
@@ -41,7 +41,8 @@ def monitor_events
 
   def queue_event(event)
     _log.info "#{log_prefix} Caught event [#{event[:name]}]"
-    EmsEvent.add_queue('add_rhevm', @cfg[:ems_id], event.to_hash)
+    event_hash = ManageIQ::Providers::Redhat::InfraManager::EventParser.event_to_hash(event, @cfg[:ems_id])
+    EmsEvent.add_queue('add', @cfg[:ems_id], event_hash)
```

Comment:
@oourfali @masayag can someone double-check this?
It seems `add_rhevm()` used to get `event.to_hash` and pass that to `event_to_hash()`, now it's `event_to_hash(event, ...` without `.to_hash`?
May be fine if it's already a hash, I'm just re-reviewing this PR as it affected our provider, and this looked worth confirming...